### ISSUE TITLE
A list of libraries in other languages?

### ISSUE THREAD

**Issue by berezovskyi** [vshymanskyy/muon#25]
Would be good to have a list in the README of known muon libraries in various programming languages.

Thanks for publishing the format, it's brilliantly simple!

**Comment by oherych** [comment]
@berezovskyi , Yes, I have Golang implementation https://github.com/oherych/muon
